src/main/java/com/ocient/cli/CLI.java

```diff
@@ -981,6 +984,33 @@ private static void outputNextQuery(final String cmd)
 		}
 	}
 
+	private static void outputSession(final String cmd) {
+		try 
+		{
+			outputSessionFile = cmd.substring("OUTPUT SESSION ".length()).trim();
+			
+			if(outputSessionFile.toLowerCase().equals("stdout")) {
+				System.setOut(new PrintStream(new FileOutputStream(FileDescriptor.out)));
```

Comment:
// When "stdout" is provided. The existing session log will be closed.
```suggestion
				System.setOut(new PrintStream(new FileOutputStream(FileDescriptor.out)));
```

Reply:
Shouldn't the PrintStream object be closed?